README.md

```diff
@@ -62,172 +53,22 @@ The new registry implementation provides the following benefits:
 
 For information on upcoming functionality, please see [ROADMAP.md](ROADMAP.md).
 
-Installation
-------------
+Deploying
+-----------
+The majority of Docker users pull images from the Docker public registry
```

Comment:
Is this statement true or do we want it to be true? Something like "docker users can use the hub but here are reasons why you may want to run your own registry". This should also link to the deployment documentation.